src/Analyzer.TemplateProcessor/ArmTemplateProcessor.cs

```diff
@@ -193,7 +193,8 @@ internal Template ProcessResourcesAndOutputs(Template template)
                     }
                     catch (Exception)
                     {
-                        logger?.LogWarning("The parsing of a template output failed. Output name: {outputName}. Output value: {outputValue}", outputKey, template.Outputs[outputKey]?.Value?.Value?.ToString());
+                        logger?.LogWarning("The parsing of a template output failed");
+                        logger?.LogDebug("Output name: {outputName}. Output value: {outputValue}", outputKey, template.Outputs[outputKey]?.Value?.Value?.ToString());
```

Comment:
Should it still say "parsing failed" or is it self-explanatory?

Reply:
How would it be self explanatory? What would you write in the warning message?

Reply:
logger?.LogDebug("The parsing of a template output failed. Output name: {outputName}. Output value: {outputValue}", outputKey, template.Outputs[outputKey]?.Value?.Value?.ToString());

Reply:
The logs would be one right next to the other, I think it looks fine like this:

![image](https://user-images.githubusercontent.com/11780216/198367798-2292458e-7a62-4182-b629-910e781ed447.png)

Reply:
Oh, I see. This makes sense, thank you. Closing